### DISCUSSION TITLE
Invalid backend response. Error occurred on the server side. HTTP error 500 due to 12 directories and 115888 files in /home/user

### DISCUSSION THREAD

**Discussion by ghost** [Studio-42/elFinder#3555] [General]
Whenever I open /home/user using elfinder, it gives:
Invalid backend response.
Error occurred on the server side.
HTTP error 500

This is happening because /home/user has 12 directories and 115888 files. The /home/user has a size of 5.5gb.
What do I need to add in the elfinder.php configuration file or any other file to solve this?

Below is $opts array in elfinder.php:
'roots' => array(
		// Items volume
		array(
			'driver'        => 'LocalFileSystem',         
			'path'          => $user['homedir'],       
			'statOwner' => true,
			'uploadAllow'    => array('all'),           
			'uploadOrder'   => array('allow', 'deny'),      
			'acceptedName' => 'validName',
			'tmbPath' => '',
			'maxArcFilesSize' => 'unlimited',
		),

**Comment by zjhappy1314** [comment]
一样的问题，不知道怎么解决。
